src/components/Button.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable import/prefer-default-export, react/prop-types */
+import { React } from 'react';
+
+export function Button(props) {
+  const { name } = props;
```

Comment:
- [ ] This prop should also have a default value of 0, just define it when you validate your props with `prop-types`.